src/stories/AxisSystem.stories.tsx

```diff
@@ -44,18 +44,49 @@ export const AspectRatio = Template.bind({});
 
 AspectRatio.args = {
   abscissaConfig: { indexDomain: [0, 10], showGrid: true },
-  ordinateConfig: { indexDomain: [0, 1], showGrid: true },
-  aspectRatio: 10,
+  ordinateConfig: { indexDomain: [0, 2], showGrid: true },
+  aspectRatio: 10 / 2,
 };
 
 export const NoGrid = Template.bind({});
 
 NoGrid.args = {
-  abscissaConfig: { indexDomain: [-5, 20] },
-  ordinateConfig: { dataDomain: [0, 1] },
+  abscissaConfig: { indexDomain: [-5, 20], showGrid: false },
+  ordinateConfig: { dataDomain: [0, 2], showGrid: false },
 };
 
+export const InheritedStyles = Template.bind({});
+
+InheritedStyles.args = {
+  abscissaConfig: { indexDomain: [0, 50], showGrid: true },
+  ordinateConfig: { indexDomain: [0, 3], showGrid: true },
+};
+
+InheritedStyles.decorators = [
+  (VisCanvasStory: Story) => (
+    <div
+      style={{
+        flex: '1 1 0%',
+        display: 'flex',
+        fontFamily: 'monospace',
+        fontWeight: 'bold',
+        fontSize: '1.125rem',
+      }}
+    >
+      <VisCanvasStory />
+    </div>
+  ),
+];
```

Comment:
I don't understand the point of this story :thinking:

Reply:
Oh, it is for styling ticks. Still it is a shame to not be able to change the styling dynamically in storybook :slightly_frowning_face: